clients/src/test/java/org/apache/kafka/clients/admin/FakeForwardingAdmin.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.clients.admin;
+
+import java.util.Map;
+
+public class FakeForwardingAdmin extends ForwardingAdmin {
```

Comment:
hmm, `FakeForwardingAdmin` is here because the `ForwardingAdmin` is in `clients/admin` but I don't mind moving them to connect/mirror at this point. they are in `clients/admin` based on some suggestions on the KIP discussion on the mailing list to have forwarding admin pluggable in other clients in the future.

Reply:
Ah, OK, that makes sense then.